qa/tasks/cephfs/fuse_mount.py

```diff
@@ -439,12 +439,21 @@ def find_socket(client_name):
             timeout=(15*60)).strip()
         log.info("Found client admin socket at {0}".format(asok_path))
 
-        # Query client ID from admin socket
-        json_data = self.client_remote.sh(
-            ['sudo', self._prefix + 'ceph', '--admin-daemon', asok_path] + args,
-            stdout=StringIO(),
-            timeout=(15*60))
-        return json.loads(json_data)
+        # Query client ID from admin socket, wait and
+        # retry 5 times if it is not ready
+        for i in range(10):
```

Comment:
````
from teuthology.contextutil import safe_while
...
  with safe_while() as proceed:
    while proceed():
      try:...
````

See also:https://github.com/ceph/teuthology/blob/ff173f9c01d947b16ae2f070e7d29a41dd875b74/teuthology/contextutil.py#L59

Reply:
Sure, I will fix it.